### PR TITLE
Changing back to Python 3.10

### PR DESCRIPTION
**Pull request by KennethSkylight**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ on:
 
 
 env:
-  TEST_RUNNER_PYTHON_VERSION: 3.11
+  TEST_RUNNER_PYTHON_VERSION: 3.10.12
```
Comment (KennethSkylight) [inline .github/workflows/test.yaml:17]: For some reason when I did just `3.10`, it gave me an error saying it tried to test with python 3.1 
So I just added the patch version so it wouldn't do that

**Review by DanPaseltiner — APPROVED** [review]
LGTM. Thanks @KennethSkylight

File: containers/alerts/requirements.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ azure-identity
 typing_extensions==4.5.0
 pydantic==1.10.9
 fastapi==0.96.0
-phdi @ git+https://github.com/CDCgov/phdi@main
+phdi @ git+https://github.com/CDCgov/phdi@python310
```
Comment (bamader) [inline containers/alerts/requirements.txt:8]: Just a flag we'll have to change this back before merging

File: containers/ingestion/requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 typing_extensions==4.6.3
 pydantic==1.10.9
 fastapi==0.96.0
-phdi @ git+https://github.com/CDCgov/phdi@main
+phdi @ git+https://github.com/CDCgov/phdi@python310
```
Comment (bamader) [inline containers/ingestion/requirements.txt:4]: Same

**Review by bamader — APPROVED** [review]
Much appreciated, Kenneth!

**Pull request merged**